src/pages/admin/AdminEvents.vue

```diff
@@ -152,23 +152,23 @@ export default {
         subaction,
         patient,
         recipient,
-        user_badge,
-        required_attr,
+        user_badge: userBadge,
+        required_attr: requiredAttr,
       } = unwrap(eventInputs);
       let data = {};
-      let badge_key = _.get(user_badge, 'key', null);
-      badge_key = badge_key === -1 ? null : badge_key;
```

Comment:
@raymondxd Oh you did end up camel-casing these?

Reply:
@BradenM Yes, I did.
I used es6 destructive code style to rename `user_badge` as `userBadge`, and made some following changes according to the variable name change